### ISSUE TITLE
Problem parsing CORA alignment file

### ISSUE THREAD

**Issue by tonyelewis** [UCLOrengoGroup/cath-tools#43]
Using this file: [1.10.287.810__FF_SSG9__1.aln_reps.cora.txt](https://github.com/UCLOrengoGroup/cath-tools/files/1333916/1.10.287.810__FF_SSG9__1.aln_reps.cora.txt)...

~~~
setenv PDBDIR /cath/data/v4_0_0/pdb
cath-superpose --cora-aln-infile 1.10.287.810__FF_SSG9__1.aln_reps.cora.txt --pdb-infile $PDBDIR/2bsk --pdb-infile $PDBDIR/2bsk --pdb-infile $PDBDIR/3dxr --pdb-infile $PDBDIR/3cjh --align-regions 'D[2bskA00]:A' --align-regions 'D[2bskF00]:F' --align-regions 'D[3dxrA00]:A' --align-regions 'D[3cjhB00]:B' --sup-to-pymol
~~~

...gives...

~~~
Cannot read CORA legacy alignment file [bad lexical cast: source type value could not be interpreted as target] : No such file or directory
2017-09-26 17:17:32.687629 [cath-superpose|error  ] Problem building alignment (and spanning tree) : Cannot read CORA legacy alignment file [bad lexical cast: source type value could not be interpreted as target]
~~~

**Comment by sillitoe** [comment]
Could you please bump the release so I can access this fix?

**Comment by tonyelewis** [comment]
@sillitoe : I've just released [v0.16.0](https://github.com/UCLOrengoGroup/cath-tools/tree/v0.16.0). CentOS6 binaries are on the GitHub release and in the usual place on our system. Travis-CI is currently working on the Ubuntu and MacOS builds.
